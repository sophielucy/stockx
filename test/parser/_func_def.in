function myfunc (int x, int y) returns int {
    3+4;
}
