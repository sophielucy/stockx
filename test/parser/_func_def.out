FUNCTION myfunc (x, y) returns int
{
	Binop(IntLiteral(3), Add, IntLiteral(4))
}

